src/common/framework/fixture.ts

```diff
@@ -237,16 +237,26 @@ export class Fixture<S extends SubcaseBatchState = SubcaseBatchState> {
   }
 
   /** Expect that the provided promise rejects, with the provided exception name. */
-  shouldReject(expectedName: string, p: Promise<unknown>, msg?: string): void {
+  shouldReject(
+    expectedName: string,
+    p: Promise<unknown>,
+    { checkForStackProperty = false, message }: ExceptionCheckOptions = {}
+  ): void {
     this.eventualAsyncExpectation(async niceStack => {
-      const m = msg ? ': ' + msg : '';
+      const m = message ? ': ' + message : '';
       try {
         await p;
         niceStack.message = 'DID NOT REJECT' + m;
         this.rec.expectationFailed(niceStack);
       } catch (ex) {
-        niceStack.message = 'rejected as expected' + m;
+        niceStack.message = 'rejected as expected, but wrong name' + m;
```

Comment:
Hmm IIUC, this is always overridden in the subsequent `expectErrorValue` call?

Reply:
You're right. We don't need this message anyway since expectErrorValue will say the same thing.

The expectationFailed below should preserve it though.